classes/class3/homework3/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please link to the git repo rather than your git profile when you submit the homework survey. Makes our lives easier.